Identifier.ts

```diff
@@ -83,46 +80,18 @@ export namespace Identifier {
 	export function previous(identifier: Identifier, decrement = 1): Identifier {
 		return next(identifier, -decrement)
 	}
-	export const length = [
-		4, 8, 12, 16, 20, 24, 28, 32, 36, 40, 44, 48, 52, 56, 60, 64, 68, 72, 76, 80, 84, 88, 92, 96, 100, 104, 108, 112,
-		116, 120, 124, 128,
-	] as Length[]
-	export type Length =
-		| 4
-		| 8
-		| 12
-		| 16
-		| 20
-		| 24
-		| 28
-		| 32
-		| 36
-		| 40
-		| 44
-		| 48
-		| 52
-		| 56
-		| 60
-		| 64
-		| 68
-		| 72
-		| 76
-		| 80
-		| 84
-		| 88
-		| 92
-		| 96
-		| 100
-		| 104
-		| 108
-		| 112
-		| 116
-		| 120
-		| 124
-		| 128
+	export type Length = typeof Length.values[number]
 	export namespace Length {
-		export function is(value: Length | any): value is Length {
-			return typeof value == "number" && value >= 4 && value <= 128 && (value & 252) == value
-		}
+		export const values = [
+			4, 8, 12, 16, 20, 24, 28, 32, 36, 40, 44, 48, 52, 56, 60, 64, 68, 72, 76, 80, 84, 88, 92, 96, 100, 104, 108, 112,
+			116, 120, 124, 128,
+		] as const
+		export const type = isly.named("cryptly.Identifier", isly.number<Length>(values as any as number[]))
```

Comment:
Should the name be `"cryptly.Identifier.Length"` instead?

Reply:
Thanks